libs/payments/ui/src/lib/client/components/SubmitButton/index.tsx

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import Image from 'next/image';
+import { useFormStatus } from 'react-dom';
+import spinnerWhiteImage from '@fxa/shared/assets/images/spinnerwhite.svg';
+import {
+  BaseButton,
+  ButtonVariant,
+} from 'libs/payments/ui/src/lib/server/components/BaseButton';
+
+export type Props = {
```

Comment:
For the props, in other components we've been using `interface` and following the naming convention of `ComponentNameProps`. Also no need to export the props.

Could this be updated to `interface SubmitButtonProps`

Reply:
Looks like some changes are still missing.